corehq/motech/repeaters/repeater_generators.py

```diff
@@ -555,7 +555,7 @@ def get_case_indices(self, target_domain, target_case, configs_by_case_id):
         if not self.index_remove_case_id:
             return indices
 
-        if self.index_remove_identifier not in indices:
+        if not target_case or self.index_remove_identifier not in indices:
```

Comment:
I don't quite follow this commit. If `target_case` is None, won't this fail? https://github.com/dimagi/commcare-hq/blob/ae8b92cbadea406a2d19af9579cb7e8dcbf92ce1/corehq/motech/repeaters/repeater_generators.py#L595

Reply:
Thanks, that should be inverted

Reply:
12f323848c8